spec/bundler/shared_helpers_spec.rb

```diff
@@ -209,6 +209,10 @@
   end
 
   describe "#set_bundle_environment" do
+    before do
+      ENV["BUNDLE_GEMFILE"] = "Gemfile"
```

Comment:
there isn't actually a gemfile there, it was just passing since there was an existing `@definition` set